application/src/main/kotlin/net/corda/v5/application/crypto/DigitalSignatureVerificationService.kt

```diff
@@ -26,4 +27,6 @@ interface DigitalSignatureVerificationService {
      * data is empty.
      */
     fun verify(publicKey: PublicKey, signatureSpec: SignatureSpec, signatureData: ByteArray, clearData: ByteArray)
```

Comment:
clearData is a terrible name. Perhaps dataBeingVerified, or originalData ?

Reply:
Changed to `originalData`. 

Also did some parameters re-ordering to hopefully make it a bit more intuitive. I understand it should be replacing the other overload so it should be ok? Otherwise I will revert it.

Reply:
Parameters order agreed offline. Also registered [CORE-9332](https://r3-cev.atlassian.net/browse/CORE-9332) to align existing overload with the one added now.